docs/quick-start.md

```diff
@@ -36,6 +36,18 @@ The second command will download and run a prebuilt AsterixDB docker container f
 After it finishes, you should see the messages as shown in the following screenshot:
 ![docker][docker]
 
+After you have installed the docker container for the first time, you may restart it later using the following command:
```

Comment:
Got it. Still a few questions.
1. Do we need to include instructions on how to shutdown docker?
2. I think 'accidentally' it not a very proper word. For example, the user may restart his machine, and after that he wants to run the TwitterMap again. Restarting the machine perhaps shouldn't be called as 'accidentally shutdown'.
What about:
"After you have installed the docker container for the first time, you can restart it using the following command in case the docker is shutdown:"